docker/ui/env-config.template.js

```diff
@@ -0,0 +1,4 @@
+window.__env__ = {
+  REACT_APP_BASE_URL: '${REACT_APP_BASE_URL}',
+  REACT_APP_ARRANGER_ADMIN_ROOT: '/${REACT_APP_BASE_URL}/admin/graphql'.replace('//', '/'),
```

Comment:
if no value is passed, this ensures the proxy remains within the host URL, rather than replacing it. (e.g. `http://localhost/admin/graphql` ✔️  vs `http://admin/graphql` ❌ )